packaging/os/yum.py

```diff
@@ -544,7 +546,7 @@ def install(module, items, repoq, yum_basecmd, conf_file, en_repos, dis_repos):
                     f.close()
                     pkg = package
                 except Exception, e:
-                    shutil.rmtree(tempdir)
+                    delete_temporary_directory(tempdir)
```

Comment:
Yeah, I was thinking the same :smile: 

Anyone else think this should be called out explicitly? I am open to discussion.